src/kafkaaggregator/config.py

```diff
@@ -71,27 +73,60 @@ class Configuration:
     workload of the application.
     """
 
-    src_topic: str = os.getenv("SOURCE_TOPIC", "kafkaaggregator-src-topic")
+    source_topic_name: str = os.getenv(
+        "SOURCE_TOPIC", "kafkaaggregator-example"
+    )
     """Name of the source topic used in the kafkaaggregator example."""
 
-    agg_topic: str = os.getenv(
-        "AGGREGATION_TOPIC", "kafkaaggregator-agg-topic"
+    topic_regex: str = os.getenv("TOPIC_REGEX", "^kafkaaggregator-example$")
+    """Regex to select topics to aggregate."""
+
+    topic_blacklist: List[str] = field(default_factory=list)
+    """Any topic names added to the topic_blacklist will not be aggregated."""
+
+    topic_rename_format: str = os.getenv(
+        "TOPIC_RENAME_FORMAT", "{source_topic_name}-aggregated"
     )
-    """Name of the aggregation topic used in the kafkaaggregator example."""
+    """A format string for the aggregation topic name, which must contain
+    ``{source_topic_name}`` as a placeholder for the source topic name."""
 
     excluded_field_names: List[str] = field(default_factory=list)
     """List of field names to exclude from aggregation."""
 
+    agents_output_dir: str = os.getenv("AGENTS_OUTPUT_DIR", "agents")
+    """Name of output directoty for the agents code."""
+
+    agents_template_file: str = os.getenv("AGENTS_TEMPLATE_FILE", "agents.j2")
```

Comment:
I've seen `.jinja` extensions but never `.j2`. I think its fine, though.

Reply:
I had this question too, Jinja2 documentation does not make any recommendation for the template file extension but I read somewhere that `.j2` is trending after its use in Ansible.